lib/context_adapter_communication.js

```diff
@@ -518,6 +534,22 @@ function getThirdPartyPayload(serviceDescriptor, operationDescriptor) {
 }
 
 /**
+ * Encodes the response sent to the Context Broker so no forbidden characters are used
+ *  The forbidden characters not allowed by the Context Broker are: < > " ' = ; ( )
+ * @param str The string to be encoded
+ * @return {string} The encoded string
+ */
+function encodeResponse(str) {
+  // encodeURIComponent() escapes all characters except the following: alphabetic, decimal digits, - _ . ! ~ * ' ( )
+  var encodedStr = encodeURIComponent(str);
```

Comment:
Well, the problem are the forbidden characters not allowed by the Context Broker and the fact that the response is dynamic (not only the errors which may be sent to the Context Broker but also the result which may come from the Third Party service) :)
We could only encode these forbidden characters and not using `encodeURIComponent()` and everything would be much more readable (in the logs :) ).
What do you think?